src/api/notification/services/notification.ts

```diff
@@ -50,14 +50,17 @@ export default factories.createCoreService(MODULE_ID, ({ strapi }) => {
     },
     async getNotificationList(account: string) {
       const push = false
+      const templateFilter = notificationsTemplateFilter(push)
       const notifications = await strapi.entityService.findMany(
         MODULE_ID,
         {
           start: 0,
           limit: 50,
           filters: {
-            account,
-            notification_template: notificationsTemplateFilter(push)
+            $or: [
+              { account, notification_template: templateFilter },
+              { account: account.toLowerCase(), notification_template: templateFilter },
```

Comment:
nit: why not just to lowecase it when we save it in the db so we don't need to implement the OR?

Reply:
It can be added from admin panel UI and we can't control the value there

Reply:
<img width="841" alt="image" src="https://github.com/cowprotocol/cms/assets/7122625/06a53e4f-1492-46c0-9c7e-43e23ee78568">

Reply:
okok